README.md

```diff
@@ -22,6 +22,33 @@ This assumes you have a `config.json` in the project directory. It can be downlo
 
 You can find the latest balenaOS images on [dockerhub](https://hub.docker.com/r/resin/resinos/tags).
 
+
+### How to use with local BoB
```

Comment:
maybe this can live in a second markdown file, since it's specific to our development workflow?